src/export/bops/tests/files.test.ts

```diff
@@ -1,15 +1,15 @@
 import type { FileTag } from "../../../types";
 import { computeBOPSParams, extractTagsFromPassportKey } from "../index";
 
-const PASSPORT_UPLOAD_KEY = "proposal.drawing.locationPlan";
+const PASSPORT_UPLOAD_KEY = "locationPlan";
```

Comment:
similarly, i'll update the `planx-data-migrations` script to additonally find/replace any DrawBoundary upload keys and open a `planx-new` pr to ensure the "default" for new components matches our new file pattern